### ISSUE TITLE
Checksum issue with v6.14.0

### ISSUE THREAD

**Issue by hbanzon** [AppsFlyerSDK/AppsFlyerFramework#280]
# Report

Trying to update from 6.13.2 to 6.14.0 but am getting the following error in Tuist

```
error: checksum of downloaded artifact of binary target 'AppsFlyerLib' (e3063f25dd58b242c31d1bc5761fcb96930539e8e4d0745c22df792ecbd455e0) does not match checksum specified by the manifest (d7fdf61996dacb7787506305b6c541197ede64fcc4528c41842d9a507bd1b94b)
```

## SDK Version

`6.14.0`

## What did you do?

`tuist fetch`

## What did you expect to happen?

Fetching of SDK is successfull

## What happened instead?

```
error: checksum of downloaded artifact of binary target 'AppsFlyerLib' (e3063f25dd58b242c31d1bc5761fcb96930539e8e4d0745c22df792ecbd455e0) does not match checksum specified by the manifest (d7fdf61996dacb7787506305b6c541197ede64fcc4528c41842d9a507bd1b94b)
```

## Please provide any other relevant information.

Happens in both Tuist 3 and Tuist 4.

**Comment by andibeqiri** [comment]
This PR could resolve this: https://github.com/AppsFlyerSDK/AppsFlyerFramework/pull/275

However the recommended solutions seems to be using static repo: https://github.com/AppsFlyerSDK/AppsFlyerFramework-Static

**Comment by iharkatkavets** [comment]
Is it hard to fix a checksum in config file for v6.14.0 ?

**Comment by iharkatkavets** [comment]
I created a pull request which fixes the issue 
https://github.com/AppsFlyerSDK/AppsFlyerFramework/pull/284

**Comment by gooson** [comment]
same issue

**Comment by Joel-Bell-okcupid** [comment]
When should we expect a new release containing the fix?

**Comment by harishvitta** [comment]
Is there any update on this ticket team?

**Comment by dayoff-dalma** [comment]
When will it be resolved?

**Comment by harrywombat** [comment]
Yep this has totally blocked my release and I can't find a workaround yet.

**Comment by xinsight** [comment]
I was getting still getting this annoying checksum issue after switching to the static library (https://github.com/AppsFlyerSDK/AppsFlyerFramework-Static). Eventually, I figured out that I also had a dependency to the old AppsFlyer in one of my modules.

My suggestion is to:

1) remove AppsFlyer from your project completely.
2) Close/Re-open Xcode and you should not see any references to AppsFlyer during Xcode's "Resolve Packages" phase. 
3) Once "Resolve Packages" does have any errors, then add: https://github.com/AppsFlyerSDK/AppsFlyerFramework-Static

**Comment by Joel-Bell-okcupid** [comment]
@xinsight Thanks for the suggestion. I attempted to use the static framework. I was able to add it as Package Dependency at the project level without issue. However, I also need to add the dependency to a swift package module within the project. I'm struggling to identify the correct syntax/formatting/declaration in the Package.swift file. Any suggestions?

**Comment by guillecom** [comment]
Also this is not a long term solution as the static library doesn't contain the privacy manifest files apparently which will be required starting may 1st 2024

**Comment by sassiwalid** [comment]
any solution please with dynamic version ?

**Comment by harrywombat** [comment]
This still isn't working for me still. I'm getting "Missing package product 'AppsFlyerLib'" now as well no matter which package version I try to add. Restarted Mac, reset package caches, deleted derived data and the rest.
